src/module.c

```diff
@@ -11486,6 +11486,14 @@ int RM_GetDbIdFromDefragCtx(RedisModuleDefragCtx *ctx) {
     return ctx->dbid;
 }
 
+char *RM_DumpACL() {
```

Comment:
I think the return type here should be an RedisString, so that the module knows that it needs to free it. The module needs some way to free the information that was passed in. This functions also need to be documented so the documentation is updated.

Reply:
changed to what I think is desired, but didn't add documentation yet.